index.html

```diff
@@ -4,21 +4,33 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
-    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="style.css">   
     <title>Hello, I am Adam Chang</title>
 </head>
 <body>
     <header class="head-container">
-        <img id="logo" src="Images/Logo.png" alt="logo" height="30"> 
-        <p>I am Adam</p>
-        <img src="Images/menu.svg" alt="">
-       
+         
+        <h2 id="name">Adam Chang</h2>
+        <img  id="menu-btn" src="Images/Union.png" alt="menu">   
 
         
     </header>
-<!-- <footer>
-    <a href='https://pngtree.com/so/red'>red png from pngtree.com/</a>
-</footer>     -->
+<section class="main-page">
+<h1 id="main-title">Hello, I am Adam Chang</h1>
+<h2 id="second-title">Welcome to my Portfolio page.</h2>
+<h4 id="description">I am a Software Developer in training</h4>
+<p id="main-p">I can build basic portfolio page, landing page, or website for your business. 
+If you are interested, lets hang out! My socials are down there!</p>
+<div class="icon-container">
+<a href="https://github.com/adamteddychang"><img src="Images/github.png" alt="github logo" height="20" class="icons"></a>
+<a href=""><img src="Images/linkedin.png" alt="linkedin logo" height="20" class="icons"></a>
+<a href=""><img src="Images/angelist.png" alt="angelist logo" height="20" class="icons"></a>
+<a href="https://twitter.com/AdamChang3"><img src="Images/twitter.png" alt="twitter logo" height="20" class="icons"></a>
+<a href=""><img src="Images/Medium.png" alt="medium logo" height="20" class="icons"></a>
+</div>
```

Comment:
The list of icons should use a `ul` tag to group them, and `li` per icon; each icon should have a `link` (a tag).

Reply:
I have wrapped the icons in <a> tags and wrapped them with <li>, hope that works, thank you.